system/lib/wasmfs/backends/node_backend.cpp

```diff
@@ -141,17 +141,18 @@ class NodeDirectory : public Directory {
 };
 
 class NodeBackend : public Backend {
-  std::string rootPath;
+  // The underlying Node FS path of this backend's mount points.
```

Comment:
```suggestion
  // The underlying Node FS path of this backend's mount point.
```

Reply:
The plural is intentional. A backend can currently be mounted arbitrarily many times.

Reply:
Ah, right... sgtm